test/iron-doc-viewer.html

```diff
@@ -34,9 +34,22 @@
       }
     </iron-doc-viewer>
 
+    <dom-module id="bound-fixture">
+      <template>
+        <iron-doc-viewer id="doc" descriptor="{{descriptor}}" bound></iron-doc-viewer>
+      </template>
+      <script>
+        window.addEventListener('WebComponentsReady', function() {
+          Polymer({
+            is: 'bound-fixture'
+          });
+        });
+      </script>
+    </dom-module>
+
     <test-fixture id="bound">
-      <template is="dom-template">
-        <iron-doc-viewer descriptor="{{descriptor}}" bound></iron-doc-viewer>
+      <template>
```

Comment:
this was implemented before dom-template was brought in. Do you still want me to change it back?